requirements.txt

```diff
@@ -10,3 +10,5 @@ sqlparse==0.4.1
 sympy==1.7
 scipy==1.6.0
 urllib3==1.26.3
+clipboard==0.0.4
+pyperclip==1.8.1
```

Comment:
Are you still using this dependency or was this left over from debugging the copy/paste testing issue?

Reply:
Pyperclip, that is

Reply:
@Yifan-Zhang618 is this dependency still needed?